spec/controllers/events_controller_spec.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe EventsController, type: :controller do
+  describe 'GET #new' do
+    it 'returns http success' do
+      get :new
```

Comment:
Can a user that is not signed in create an event? Before a user can create an event, it needs to sign in. You should have two tests in here: one test that fails to create an event (or get the :new) page if the user isn't signed in., and one that creates an event when a user is signed in. This test in particular **should** fail because this user hasn't signed in.

[`REQUIRED`]